addons/mail/wizard/mail_resend_message_views.xml

```diff
@@ -0,0 +1,42 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+    <data>
+        <record id="mail_resend_message_view_form" model="ir.ui.view">
+            <field name="name">mail.resend.message.form</field>
+            <field name="model">mail.resend.message</field>
+            <field name="groups_id" eval="[(4,ref('base.group_user'))]"/>
+            <field name="arch" type="xml">
+                <form string="Edit Partners">
+                    <field name="mail_message_id" invisible="1"/>
+                    <field name="notification_ids" invisible="1"/>
+                    <field name="has_cancell" invisible="1"/>
+                    <p>Please, select the action to do on each email and correct the email if needed. The modified email will be saved on partner</p>
+                    <field name="partner_ids">
+                        <tree string="Recipient" editable="top" create="0" delete="0">
+                            <field name="partner_readonly" invisible="1"/>
+                            <field name="name" readonly="1"/>
+                            <field name="email" attrs="{'readonly': [('partner_readonly', '=', True)]}"/>
+                            <field name="message" readonly="1"/>
+                            <field name="resend" widget="boolean_toggle"/>
+                        </tree>
+                    </field>
+                    <p attrs="{'invisible':[('has_cancell', '==', False)]}">Caution: It won't be possible to send this message again to partner not marked as send.</p>

```

Comment:
_Caution: It won't be possible to send this mail again to the recipients you did not select_

+ maybe add fa-warning / fa-info-circle?